### ISSUE TITLE
[E2B-457] Unable to create custom templates with Dockerfiles containing more than ~4.3GB of local data

### ISSUE THREAD

**Issue by mlejva** [e2b-dev/E2B#278]
When a user is creating a custom template, they specify a Dockerfile. If the Dockerfile requires to copy more than 4,294,967,295 bytes of a local data, it will fail with the following message

```
Preparing sandbox template building (35639 files in Docker build context).
Found ./e2b.Dockerfile that will be used to build the sandbox template.
node:internal/blob:165
      throw new ERR_BUFFER_TOO_LARGE(0xFFFFFFFF);
      ^
RangeError [ERR_BUFFER_TOO_LARGE]: Cannot create a Buffer larger than 4294967295 bytes
    at new NodeError (node:internal/errors:399:5)
    at new Blob (node:internal/blob:165:13)
    at R1.<anonymous> (/Users/default/Library/pnpm/global/5/.pnpm/@e2b+cli@0.1.20_openai@4.24.1/node_modules/@e2b/cli/dist/index.js:104:1093)
    at R1.emit (node:events:513:28)
    at n1.updateNonPrimary (/Users/default/Library/pnpm/global/5/.pnpm/@e2b+cli@0.1.20_openai@4.24.1/node_modules/@e2b/cli/dist/index.js:54:11372)
    at n1.update (/Users/default/Library/pnpm/global/5/.pnpm/@e2b+cli@0.1.20_openai@4.24.1/node_modules/@e2b/cli/dist/index.js:54:11193)
    at n1.ez (/Users/default/Library/pnpm/global/5/.pnpm/@e2b+cli@0.1.20_openai@4.24.1/node_modules/@e2b/cli/dist/index.js:54:13685)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11) {
  code: 'ERR_BUFFER_TOO_LARGE'
}
Node.js v18.16.1
```

We're currently using Node's `Blob` to represent the files and upload them all at once. One of the potential solutions to this issue is to start streaming the data in chunks to the backend.

<sub>From [SyncLinear.com](https://synclinear.com) | [E2B-457](https://linear.app/e2b/issue/E2B-457/unable-to-create-custom-templates-with-dockerfiles-containing-more)</sub>

**Comment by mlejva** [comment]
The maximum sizes of a Buffer in Node is described [here](https://nodejs.org/api/buffer.html#bufferconstantsmax_length).

> On 32-bit architectures, this value currently is 230 - 1 (about 1 GiB).
> On 64-bit architectures, this value currently is 232 (about 4 GiB).

**Comment by jakubno** [comment]
Solved by fc037a2b
